File: Domain/Models/Donation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Models
{
    public class Donation
    {
        public int ID { get; set; }
        public string DonorName { get; set; }
        public string Email { get; set; }
        public decimal Amount { get; set; }
        public DateTime DonationDate { get; set; }
        public string PaymentId { get; set; }
    }
}

File: WWWRubberCityFoundation/src/app/models/donation.ts
export class Donation {
  id: number;
  donorName: string;
  email: string;
  amount: number;
  donationDate: Date;
  PaymentId: string;

  constructor(
    id: number,
    donorName: string,
    email: string,
    amount: number,
    donationDate: Date,
    PaymentId: string
  ) {
    this.id = id;
    this.donorName = donorName;
    this.email = email;
    this.amount = amount;
    this.donationDate = donationDate;
    this.PaymentId = PaymentId;
  }
}
